### PR TITLE
custom chart tooltips

### PR DESCRIPTION
**Pull request by jmensch1**
created custom tooltips for the charts
replaced react-tooltip with custom one in HoverOverInfo

### CODE REVIEW

**Comment by sellnat77** [comment]
Pretty cool beta feature at the bottom that shows warnings, do we wanna create an issue to address this? custom alert vs removing altogether

https://github.com/hackforla/311-data/pull/357/files#file-src-components-main-menu-dateselector-daterangepicker-jsx-L186

File: src/components/Visualizations/TotalRequests.jsx [hunk 1]
```diff
@@ -65,13 +66,33 @@ const TotalRequests = ({
...
-          const total = data.datasets.reduce((p, c) => p + c.data[index], 0);
-          return `Total ${total}`;
-        },
-      },
+      custom: adapter(ttData => {
+        const lines = [];
+        let total = 0;
+
+        ttData.body.forEach((line, idx) => {
+          const [abbrev, requests] = line.lines[0].split(': ');
```
Comment (sellnat77) [inline src/components/Visualizations/TotalRequests.jsx]: This looks wild... can you drop some comments in here to 'splain whats going on?
Or even extract this to its own function if its something that will be modified often

Comment (jmensch1) [reply src/components/Visualizations/TotalRequests.jsx]: Yeah, I know it's a bit ugly...basically for every tooltip we need to convert ChartJS's tooltip data model into some that our own custom tooltip can digest. And their model is pretty awkward. But yeah I can put some comments in there.

**Comment by jmensch1** [comment]
I'm gonna close this for now, will be better to handle a bit later on.

**Pull request closed without merging**